src/pfb/importers/gen3dict.py

```diff
@@ -275,7 +275,10 @@ def _array_type(property_type):
         if "description" in property_type:
             enum["name"] = property_type["description"]
         else:
-            enum["name"] = property_type["termDef"][0]["term"]
+            enum["name"] = property_type["term"]["termDef"]["term"]
```

Comment:
This should exist but should we do something safer like `property_type.get("term", None).get("termDef").get("term")` so we know we get a `None` returned

Reply:
defaulting to None will still be unsafe. If you want to be safe I believe the format would be
`result = property_type.get("term", {}).get("termDef", {}).get("term")`